examples/with-external-scoped-css/static/css/bundle.css

```diff
@@ -0,0 +1,4 @@
+.index__x___ASWLj {
+    font-size: 26px;
+    color: #238499;
+}
```

Comment:
It seems the point of the example is that this `bundle.css` is auto-generated, isn't it? If so, this file should not be committed to the repo. Is this the case?